src/parsers/srdf.hpp

```diff
@@ -94,6 +95,22 @@ namespace pinocchio
                                     const std::string & filename,
                                     const bool verbose = false) throw (std::invalid_argument)
     { return getNeutralConfiguration(model,filename,verbose); }
+
+
+    ///
+    /// \brief Get the reference configurations of a given model associated to a SRDF file.
+    ///        It throws if the SRDF file is incorrect. The reference configurations are
+    ///        saved in a map indexed by the configuration name
```

Comment:
Can you add missing points in the documentation?

Reply:
What do you mean by missing points?

Reply:
To close the sentences of your phrases. There are two missing points.
I know, this is a bit fullish, but it will avoid additional work later.

Reply:
Done.